README.md

```diff
@@ -288,3 +288,39 @@ type BoltLogger interface {
 
 For a customised logging target, you can implement the above interface and pass an instance of that implementation to the `BoltLogger` field.
 
+## Developing
```

Comment:
How about using `CONTRIBUTING.md` for this instead?
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#about-contributing-guidelines

Reply:
```suggestion
## Development
```

Reply:
All the other drivers have building and development instructions in the Readme. Doing it for the go driver makes it more consistent with the other repos.